model_analyzer/config/generate/generator_utils.py

```diff
@@ -116,3 +116,18 @@ def generate_doubled_list(min_value: int, max_value: int) -> List[int]:
             list.append(val)
             val *= 2
         return list
+
+    @staticmethod
+    def extract_max_tokens_from_request_parameter(request_parameter: str) -> int:
```

Comment:
Can we make it more generic so that we don't need to create method for each parameter?

Reply:
Done